cypress/e2e/tests/administration/credentials/jira_crud.test.ts

```diff
@@ -0,0 +1,40 @@
+import { login } from "../../../../utils/utils";
+import { getRandomCredentialsData } from "../../../../utils/data_utils";
+import { CredentialType } from "../../../types/constants";
+import { CredentialsJira } from "../../../models/administration/credentials/credentialsJira";
+
+describe(["@tier2"], "Validation of jira credentials", () => {
+    const toBeCanceled = true;
+    const validJiraCredentials = getRandomCredentialsData(CredentialType.jira, "", true);
+    const randomJiraCredentials = getRandomCredentialsData(CredentialType.jira, "", false);
+    const jiraCredentials = new CredentialsJira(randomJiraCredentials);
+
+    before("Login", function () {
+        // Perform login
+        login();
+    });
+
+    it("Creating Jira credentials and cancelling without saving", () => {
+        jiraCredentials.create(toBeCanceled);
+    });
+
+    it("Creating Jira credentials", () => {
+        jiraCredentials.create();
+    });
+
+    it("Editing Jira credentials and cancelling without saving", () => {
```

Comment:
Please add the bug number in the TC , so that we know in jenkins run why it is failing .

Example 
it("Editing Jira credentials and cancelling without saving- Failing due to Bug MTA -38",

Reply:
Done